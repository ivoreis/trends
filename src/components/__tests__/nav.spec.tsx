import React from 'react'
import renderer from 'react-test-renderer'
import Nav from '../nav'

test('renders with correct styles', () => {
  const tree = renderer.create(<Nav />).toJSON()
  expect(tree).toMatchSnapshot()
})
