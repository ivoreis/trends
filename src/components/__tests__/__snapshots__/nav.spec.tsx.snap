// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with correct styles 1`] = `
.emotion-0 nav {
  text-align: center;
}

.emotion-0 ul {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0 32px;
}

.emotion-0 nav > ul {
  padding: 4px 16px;
}

.emotion-0 li {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 6px 8px;
}

.emotion-0 a {
  color: #0756a7;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 13px;
}

<nav
  className="emotion-0"
>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="https://zeit.co/now"
      >
        ZEIT
      </a>
    </li>
    <li>
      <a
        href="https://github.com/zeit/next.js"
      >
        GitHub
      </a>
    </li>
  </ul>
</nav>
`;
